spk/flexget/src/requirements.txt

```diff
@@ -1,33 +1,36 @@
-paver
-FeedParser>=5.1.3
-argparse
-babelfish>=0.5.4
-stevedore>=0.14
-BeautifulSoup
-beautifulsoup4>=4.1, !=4.2.0, <4.4
-plumbum
-rpyc
-path.py
+FeedParser>=5.2.1
```

Comment:
So my initial fear about dependency management was right... Please let me a couple of days to learn how to get it right.